views/partials/login.ejs

```diff
@@ -13,5 +13,5 @@
 </form>
 
 <div class="container">
-  <button id="signup-action" class="btn btn-warning">Sign up</button>
+  <button id="logout-action" class="btn btn-warning">Sign up</button>
```

Comment:
Bear in mind, we might need to turn the english labels into Chinese, is it going to be a lot of work since labels are hardcoded? 

Is this ready to merge?